socorro/processor/mozilla_transform_rules.py

```diff
@@ -1088,3 +1088,47 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
             processed_crash
         )
         return True
+
+
+#==============================================================================
+class ThemePrettyNameRule(Rule):
+    """The Firefox theme shows up commonly in crash reports referenced by its
+    internal ID. The ID is not easy to change, and is referenced by id in other
+    software.
+
+    This rule attempts to modify it to have a more identifiable name, like
+    other built-in extensions.
+
+    Must be run after the Addons Rule."""
+
+    #--------------------------------------------------------------------------
+    def __init__(self, config):
+        super(ThemePrettyNameRule, self).__init__(config)
+        self.conversions = {
+            "{972ce4c6-7e08-4474-a285-3208198ce6fd}":
+                "{972ce4c6-7e08-4474-a285-3208198ce6fd} "
+                "(default Firefox theme)",
+        }
+
+    #--------------------------------------------------------------------------
+    def version(self):
+        return '1.0'
+
+    #--------------------------------------------------------------------------
+    def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
+        '''addons is a list of tuples containing (extension, version)'''
```

Comment:
I went with a docstring and a default value instead of the explicit test because it illustrates what we expect when we get the addons object better and provides a similar level of defense